files/tests/fixture_accounts.py

```diff
@@ -0,0 +1,49 @@
+
+from . import util
+
+from bs4 import BeautifulSoup
+from files.__main__ import app
+import pytest
+from time import time, sleep
+
+class AccountsFixture:
+	accounts = {}
+
+	def account(self, name = "default"):
```

Comment:
use the `@lru_cache` annotation would be a little simpler than maintaining a dict here

Reply:
oh nice, actual Python knowledge :D

Yeah, good catch. Change made!